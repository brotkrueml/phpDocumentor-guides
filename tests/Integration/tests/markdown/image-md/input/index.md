# Markdown Image

![Hero Illustrations](hero-illustration.svg)
