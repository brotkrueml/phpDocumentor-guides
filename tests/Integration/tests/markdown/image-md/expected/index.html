<!DOCTYPE html>
<html lang="en">
<head>
    <title>Markdown Image - Project Title</title>

</head>
<body>
    <h1>Markdown Image</h1>

<p><img src="hero-illustration.svg" alt="Hero Illustrations"/></p>
</body>
</html>
